dts/msm8916/msm8939-mtp.dts

```diff
@@ -16,8 +16,22 @@
 	};
 
 	xiaomi-ferrari {
+		/* Included becuase log said qcom,msm-id entry not found and this made it recognize the device tree correctly */
+		qcom,msm-id = <239 0>;
+		qcom,board-id = <8 0>;
```

Comment:
This shouldn't be necessary... Does the display show an error? Don't worry about the log.

Reply:
Ok. The display works fine. It was just log.